docs/endaq/quickstart.rst

```diff
@@ -98,99 +98,40 @@ first.
-   print("Device Serial Number:", dev.serial)
-   print("Device Hardware Version:", dev.hardwareVersion)
-   print("Device Firmware Version:", dev.firmwareVersion)
-   print("Device Channels:", dev.channels)
-   # To view all possible properties to access, look at the "The Recorder Class"
-   # section of the documentation.
-
-   # Some basic configurations:
-   print("\nCONFIGURATION")
-   print("Device ready for configuration:", dev.config.available)
-   dev.config.enableChannel(dev.channels[80], False)  # Disable Ch 80
-   dev.config.enableChannel(dev.channels[80])
-   dev.config.setSampleRate(dev.channels[80], 1000)  # Set Ch 80 SR to 1000 Hz
-   print("New Channel 80 Sample Rate:",
-         dev.config.getSampleRate(dev.channels[80]))
-   dev.config.revert()  # Revert to default original configuration
-   print("Original Channel 80 Sample Rate:",
-         dev.config.getSampleRate(dev.channels[80]))
-   print("Settable triggers for Channel 80:",
-         dev.config.getTrigger(dev.channels[80]))
-   print("Settable triggers for all channels:", dev.config.getTriggers())
-   dev.config.setTrigger(dev.channels[80], high=20)
-   dev.config.applyConfig()
-   # To view all possible configurations, look at the "Configuration and Control"
-   # section of the documentation.
-
-   # Some basic commands:
-   # Note: Commands can only be sent to a physical enDAQ device;
-   # They will not work on Virtual Devices (imported .IDE files).
-   print("\nCOMMANDS")
-   print("Device ready for commands:", dev.command.available)
-   print("Device Battery Status:", dev.command.getBatteryStatus())
-   print("Device Clock Drift:", dev.command.getClockDrift())
-   print("Device Time:", dev.command.getTime())
-   dev.command.setTime()  # Set the device time to the computer's time
-   dev.command.blink()  # Blink the device's LEDs
-   # 'blink()' is helpful for identifying a specific device when multiple devices
-   # are plugged in to one computer
-   commandWait(10)  # Defined above! Waits and prints useful information.
+   # Update configuration
+   dev.config.setSampleRate(dev.channels[80], 4000)  # Set Ch 80 SR to 4000 Hz
+   dev.config.retrigger = False # Turn off retrigger
+   if dev.firmwareVersion < 30106:
+      dev.config.recordingTimeLimit = 30 # Set recording limit to 30s for old FW
+
+   # Start Recording
    dev.command.startRecording()
-   commandWait(10)
-   print("Recording stopped:", dev.command.stopRecording())
-   # To view all possible commands, look at the "Configuration and Control" section
-   # of the documentation.
-
-   # Virtual Devices:
-   # Using a recording file to create a 'virtual' version of the recorder that created it.
-   # Useful for discovering device information but can't be used to run commands.
-   with openFile('test.ide') as doc:
-      virtual_dev = endaq.device.fromRecording(doc)
+
+   # For older devices without a SerialCommandInterface
+   if "SerialCommandInterface" not in str(dev.command):
```

Comment:
This looks like a nice simple way to check, but I'm not very familiar with it, my questions may be way off base:
Is `SerialCommandInterface` a property? If it does exist, do we need to make sure `dev.command.SerialCommandInterface == True`? The S3-D16 may have `dev.command.SerialCommandInterface == False`, and I think older devices won't have `SerialCommandInterface` at all